Resources/public/js/views/ez-subitemlistview.js

```diff
@@ -284,6 +455,16 @@ YUI.add('ez-subitemlistview', function (Y) {
              * @type Array of {Object} array containing location structs
              */
             subitems: {},
+
+            /**
+             * Boolean to check if a subitem priority is currently selected.
+             *
+             * @default false
+             * @type Boolean
+             */
+            isOnePrioritySelected: {
```

Comment:
I don't really see the point of adding this attribute but maybe I miss something ?